### PR TITLE
Timelapse 4th Update- React

### PR DESCRIPTION
**Pull request by rachelcynthia**
This is the outline of how the webpage will be.
The screenshot of current web page is attached.
Screenshot (88)

This has only the header completely defined.
Everything else will be updated in future updates.

### CODE REVIEW

**Comment by bhavaniravi** [comment]
Merging this PR since it got basic elements for future contributors. 
 
Next steps 
1. Get the functionality in
2. Match the UI exactly to the design
3. Check for responsiveness

File: src/components/DateTime.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        color:"white",
+        backgroundColor:"black"
+      };
+      const day=["Sun","Mon","Tue","Wed","Thu","Fri","Sat"];
+      const month=["January", "February","March","April","May","June","July","August","September","October","November","December"];
+      const theDate=this.state.date.getDate();
+      return(
+        <div style={thisStyle}>
+          <div style={{textAlign:"left", position:"absolute",paddingLeft:"20px"}}>Timelapse</div>
+            {theDate}{this.next(theDate)}&nbsp;&nbsp;
```
Comment (bhavaniravi) [inline src/components/DateTime.jsx:48]: Remove all nbsp, all spacing should be done with margins and padding

File: src/logo.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (bhavaniravi) [inline src/logo.svg:1]: Remove this file

File: src/serviceWorker.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```
Comment (bhavaniravi) [inline src/serviceWorker.js:1]: I don't think we need service worker remove the same

File: src/components/Other/Plus.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+class Plus extends React.Component{
+    render(){
+        return(
+                <div style={{display: "inline"}}>&nbsp;Plus&nbsp;</div>
```
Comment (bhavaniravi) [inline src/components/Other/Plus.jsx:5]: Same as above no $nbsp

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+            <Equal />
+            <Card 
+              type={1}
+              date= {this.state.date}
+              days= {this.state.days}
+            />
+        </div>
+        {console.log(this.state.date)}
+        <div className="box">
+          <input
```
Comment (bhavaniravi) [inline src/App.js:55]: For the next PR make all the date pickers one component and resue it

File: src/components/DateTime.jsx (hunk 1 above)
Comment (rachelcynthia) [reply src/components/DateTime.jsx:48]: okay mam

File: src/App.js (hunk 5 above)
Comment (rachelcynthia) [reply src/App.js:55]: yes okay mam

**Comment by rachelcynthia** [comment]
Have updated the requested changes.

File: src/components/DateTime.jsx (hunk 1 above)
Comment (bhavaniravi) [inline src/components/DateTime.jsx]: Remove nbsp

**Pull request merged**